tests/experiments/experiment_unittests.py

```diff
@@ -340,17 +374,20 @@ def run_tests(raw_args=None, buffer=True, print_results=True):
         "--no-tests",
         required=False,
         action="store_true",
-        help="Only test the main experiments, not those in testing_archive/",
+        help="Only test the main experiments, not those in tests/",
+    )
+    parser.add_argument(
+        "--verbosity", required=False, default=1, help="Verbosity level, larger number increases verbosity."
     )
     args = parser.parse_args(raw_args)
 
     os.environ["RADAR_ID"] = args.site_id
 
```

Comment:
This change didn't work on the lab computer - if I specify a site ID via the command options, the script continues to run with whatever my original RADAR_ID environment variable was (it doesn't seem to change the environment variable successfully).

Can we run tests for a specific site without changing the environment variable?